### PR TITLE
Auto-Retry during import data in case of COPY command failures

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

File: yb-voyager/cmd/constants.go [hunk 1]
```diff
@@ -32,6 +32,8 @@ const (
 	POSTGRESQL                  = "postgresql"
 	LAST_SPLIT_NUM              = 0
 	SPLIT_INFO_PATTERN          = "[0-9]*.[0-9]*.[0-9]*.[0-9]*"
+	COPY_AUTORETRY_COUNT        = 5
```
Comment (amit-yb) [inline yb-voyager/cmd/constants.go:35]: Drop the word "AUTO" from everywhere.

Just COPY_MAX_RETRY_COUNT is sufficient.

File: yb-voyager/cmd/importData.go [hunk 2]
```diff
@@ -832,16 +833,22 @@ func doOneImport(task *SplitFileImportTask, connPool *tgtdb.ConnectionPool) {
 			utils.ErrExit("rename %q to %q: %s", task.SplitFilePath, inProgressFilePath, err)
 		}
 
+		copyCommand := getCopyCommand(task.TableName)
```
Comment (amit-yb) [inline yb-voyager/cmd/importData.go:836]: yb-voyager must exit if `copyCommand == ""`.

**Review by amit-yb — COMMENTED** [review]
The patch looks good. Just make voyager exit if copyCommand is empty string. After that I will approve the PR.

File: yb-voyager/cmd/constants.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/cmd/constants.go]: done

**Review by amit-yb — APPROVED** [review]
Please file a ticket to track the change that you will do to exit when `copyCommand == ""`.

**Pull request merged**